RecoTracker/MeasurementDet/plugins/MeasurementTrackerEventProducer.cc

```diff
@@ -49,6 +60,9 @@ MeasurementTrackerEventProducer::produce(edm::Event &iEvent, const edm::EventSet
     edm::ESHandle<MeasurementTracker> measurementTracker;
     iSetup.get<CkfComponentsRecord>().get(measurementTrackerLabel_, measurementTracker);
 
+    edm::ESHandle<SiPixelFedCablingMap> cablingMap;
+    iSetup.get<SiPixelFedCablingMapRcd>().get(pixelCablingMapLabel_, cablingMap);
```

Comment:
To answer myself, apparently the cabling map label is currently an empty string for both offline and HLT (at least what concerns the configurations of `SiPixelRawToDigi`), so in practice the call succeeds (but I'd still fetch it only if `!theBadPixelFEDChannelsLabels.empty()`).